src/components/icon/icon.js

```diff
@@ -55,6 +60,116 @@ const IconType = {
  * @extends {Component}
  */
 class Icon extends Component {
+  defaultColor: string = style.white;
+  activeColor: string = style.brandColor;
+  className: string = '';
+
+  /**
+   * @constructor
+   * @param {Object} props - component props
+   */
+  constructor(props: Object) {
+    super(props);
+    if (props.path) {
+      const {id} = props;
+      this.className = `playkit-icon-${id}`;
```

Comment:
id could be `undefined`?

Reply:
It can be `undefined` like any other prop can.
Are u suggest to add protection for it?

Reply:
yes